bot/slack/slack_bot.py

```diff
@@ -0,0 +1,161 @@
+import random
+import sys
+import os
+import time
+
+from slackclient import SlackClient
+
+from bot_plugin_set import BotPluginSet
+
+from mozdef_util.utilities.logger import logger
+
+
+greetings = [
+    "mozdef bot in da house",
+    "mozdef here..what's up",
+    "mozdef has joined the room..no one panic",
+    "mozdef bot here..nice to see everyone"
+]
+
+
+def formatAlert(alert_dict):
```

Comment:
Function names should be in `snake_case` as per [PEP 8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).

Reply:
Fixed in https://github.com/mozilla/MozDef/pull/1018/commits/46b9e7e0650f7032997e07369c724098375fa02a